index.html

```diff
@@ -22,11 +28,14 @@ <h2>Experience</h2>
       </ul>
     </section>
 
-    <section id = "connect">
+    <section id = "connect" >
 <h2>Connect</h2>
-<a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a>
-<a href="gmail.com"> gmail</a>
-<a href="instagram"> Instagram</a>
+
+
+<li><a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a></li>
+<li><a href="gmail.com"> gmail</a></li>
+<li><a href="instagram"> Instagram</a></li>
```

Comment:
You'll want to change the style of these connect links in the CSS to meet the assignment goals. Changing the font size, color when they're hovered over, or font style would all work.